reyna-test/src/com/b2msolutions/reyna/DispatcherTest.java

```diff
@@ -100,7 +102,7 @@ public void sendMessageHappyPathShouldSetExecuteCorrectHttpPostAndReturnOK() thr
 
     @Test
     public void sendMessageHappyPathWithChineseCharactersShouldSetExecuteCorrectHttpPostAndReturnOK() throws URISyntaxException, IOException, KeyManagementException, UnrecoverableKeyException, KeyStoreException, NoSuchAlgorithmException, CertificateException {
-        Message message = RepositoryTest.getMessageWithHeaders("谷歌拼音输入法");
+        Message message = RepositoryTest.getMessageWithHeaders("谷歌拼音输入");
```

Comment:
Moving back to the old version. This was failing due to the missing check for the shouldGzip function. With the check in place the previous test case works. I didn't observe that this test sets up for no compression headers (getMessageWithHeaders instead of getMessageWithGzipHeaders). Nice spot.